welcome.php

```diff
@@ -0,0 +1,99 @@
+
+<?php 
+include('server.php');
+  if (!isset($_SESSION['username'])) {
+  	$_SESSION['msg'] = "You must log in first";
+  	header('location: login.php');
+  }
+  if (isset($_GET['logout'])) {
+    session_unset();
+    session_destroy();
+    header("location: welcome.php");
+  }
+  
+  if (isset($_GET['delete'])) {
+    $temp = $_SESSION['id'];
+    $sql = "DELETE FROM userData where userID = $temp;";
+    mysqli_query($db, $sql);
+    session_destroy();
```

Comment:
Shouldn't there be a session_unset here as well?

Reply:
I'll just add it